x-pack/plugin/esql/qa/server/single-node/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/PushExpressionToLoadIT.java

```diff
@@ -335,36 +335,6 @@ public void testMvMaxToDouble() throws IOException {
         );
     }
 
-    public void testVCosine() throws IOException {
```

Comment:
Vector similarity functions tests have been moved to `PushVectorSimilarityToLoadIT`, as they don't need the check now. We should merge them together again when pushing down is not behind snapshot.